apps/website/src/components/documentation/tsdoc/TSDoc.tsx

```diff
@@ -56,6 +64,24 @@ export function TSDoc({ item, tsdoc }: { readonly item: ApiItem; readonly tsdoc:
 
 						if (!foundItem && !resolved) return null;
 
+						if (resolved && resolved.package === 'discord-api-types') {
+							const { displayName, kind, members, containerKey } = resolved.item;
+							let href = DISCORD_API_TYPES_DOCS_URL;
+
+							// dapi-types doesn't have routes for class members
+							// so we can assume this member is for an enum
+							if (kind === 'enum' && members[0]) href += `/enum/${displayName}#${members[0].displayName}`;
+							else if (kind === 'type' || kind === 'var') href += `#${displayName}`;
+							else href += `/${mapKindToMeaning(kind)}/${displayName}`;
```

Comment:
Is there no way we could consolidate this logic used in ExcerptText.tsx?

Reply:
we could make it a util function, but their used values as so different in a lot of their properties that it would become a function with many arguments but not much logic inside.